src/pages/config/config-details-changes.jsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from "react";
+import { useParams, useOutletContext } from "react-router-dom";
+
+import { getConfigChange, getConfig } from "../../api/services/configs";
+import { toastError } from "../../components/Toast/toast";
+import { ConfigChangeHistory } from "../../components/ConfigChangeHistory";
+import { BreadcrumbNav } from "../../components/BreadcrumbNav";
+
+export function ConfigDetailsChangesPage() {
+  const { id } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Can we use loading states like idle, loading, loaded here.

Reply:
Yes. Let's pick it up as a separate MR? E.g. useLoading.

Reply:
https://github.com/orgs/flanksource/projects/3#card-82549881 For tracking.